src/pd/pd.rs

```diff
@@ -558,15 +558,15 @@ impl<T: PdClient> Runnable<Task> for Runner<T> {
                     handle,
                     region,
                     peer,
-                    RegionStat::new(
-                        down_peers,
-                        pending_peers,
-                        written_bytes_delta,
-                        written_keys_delta,
-                        read_bytes_delta,
-                        read_keys_delta,
-                        approximate_size,
-                    ),
+                    RegionStat {
```

Comment:
Unrelated changes.

Reply:
I prefer keep the new code because add new arguments to `RegionStat::new` will cause `TooManyArguments` lint warning.

Reply:
I suspect you can do `#[allow(TooManyArguments)]` on this block if that's the case.